tools/SourceKit/bindings/python/sourcekitd/__init__.py

```diff
@@ -21,4 +21,4 @@
     Bindings for the sourcekitd framework.
 """
 
-__all__ = ['capi', 'request']
+__all__ = ('capi', 'request')
```

Comment:
All of these transformations to `__all__` are probably incorrect, no? As far as I know, `__all__` is a **list**, not a tuple.